README.md

```diff
@@ -18,7 +18,9 @@ npx @gooddata/create-gooddata-react-app my-app
 
 Then follow the instructions provided by the CLI.
 
-_NOTE:_ We use `yarn` dependency manager. To install it, follow its [documentation](https://yarnpkg.com/lang/en/docs/install). If you don't want to use `yarn`, provide the `--no-install` flag like `npx @gooddata/create-gooddata-react-app my-app --no-install` . This will skip calling `yarn install` and you will have to manually install the dependencies of the created app. Please note that we do not provide `package-lock.json` so the application may not work properly when dependencies are installed using `npm`.
+When navigating to your application using an IP address instead of localhost, expect running into `401` errors while trying to login. This is due to proxying and cookie rewriting rules, thatthe generated application uses to get around CORS. Please use localhost instead of IP to access the application because the dev server rewrites GDC cookies to be for localhost.
```

Comment:
When navigating to your application using an IP address instead of localhost, you may run into `401` errors while trying to log in. This is due to proxying and cookie rewriting rules that the generated application uses to get around CORS. Use localhost instead of IP addresses to access the application because the development server rewrites GDC cookies to be for localhost.